api/proto/teleport/transport/v1/transport_service.proto

```diff
@@ -147,6 +147,8 @@ message ProxyWindowsDesktopSessionResponse {
   // A chunk of data from the connection. Can be empty (for example, to send a message
   // signaling a successful connection even if there's no data available in the connection).
   bytes data = 1;
+  // Windows Desktop Service version
+  string version = 2;
```

Comment:
Hmm, this is adding a version field to every single message that flows from the TDP stream (agent) to the Teleport Connect UI.

This will be a bit wasteful, since once we've established a connection the version is never going to change - we'd be sending the same value over and over.

Reply:
This is a much simpler solution, my thinking was flawed because I thought Connect pulled a user's settings from the cluster and used them.